config/software/supermarket.rb

```diff
@@ -15,7 +15,7 @@
 #
 
 name "supermarket"
-default_version "master"
+default_version "irving/remove_ruby_pin"
```

Comment:
This should be changed back once your changes have been merged into Supermarket.